libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/RepeatDialog.cs

```diff
@@ -26,6 +26,18 @@ public RepeatDialog(object options = null, [CallerFilePath] string callerPath =
             this.RegisterSourceLocation(callerPath, callerLine);
         }
 
+        /// <summary>
+        /// Gets or sets an optional expression which if is true will allow loop of the repeated dialog.
+        /// </summary>
+        /// <example>
+        /// "user.age > 18".
+        /// </example>
+        /// <value>
+        /// A boolean expression.
+        /// </value>
+        [JsonProperty("allowLoop")]
+        public BoolExpression AllowLoop { get; set; }
```

Comment:
>AllowLoop [](start = 30, length = 9)

You need to update Microsoft.RepeatDialog.schema with this property.

Reply:
Updated~